SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -412,8 +422,16 @@ UnitTestResult {
 		if (message.size > 0) {
 			Post << " - " << message;
 		};
-		if (brief.not && details.notNil) {
-			Post << Char.nl << details;
+		if (brief.not) {
+			if (debug.size > 0) {
+				Post << Char.nl << debug;
+
+				// ensure debug is only output once
+				testClass.resetDebug;
```

Comment:
Does this have the effect of clearing debug messages that would occur if a later assert in the test failed? Not sure I understand this

Reply:
This was to work around the lack of isolation, but #3836 fixed that so I've dropped it.